pages/guides/business-verified-customer/frequently-asked-questions.mdx

```diff
@@ -8,66 +8,245 @@ meta:
-
-#### **Q: My Customer needs to send more funds than is allowed in my Dwolla services agreement. What is the best way to go about this?**
-
-- Your Customer is able to send multiple separate transfers as long as each transfer amount is less than the transfer limit defined in your services agreement.
-
-  > Example Scenario: The transaction limit for my Customer is $10,000 and they need to send $15,000. In this case, you can prompt your Customer to send two transfers. One for $10,000 and another for $5,000.
+<Collapsible
+  triggerText={
+    <span>
+      Q: My Customer has a <code>retry</code> status. What activity would they
+      be able to engage in while being in <code>retry</code> status, as it
+      relates to the Dwolla Platform?
+    </span>
+  }
+>
+  <ul>
+    <li>Send funds - No</li>
+    <li>
+      Receive funds - Yes - Note that funds will only process to their balance
+      and the transfer will stay <code>pending</code> until the Customer has
+      been verified.
+    </li>
+    <li>Add and verify a bank funding source - Yes</li>
+    <li>Add and verify a Beneficial Owner - Yes</li>
+  </ul>
+</Collapsible>
+
+<br />
+<br />
+
+<Collapsible
+  triggerText={
+    <span>
+      Q: My Customer has a <code>document</code> status. What activity would
+      they be able to engage in while being in <code>document</code> status, as
+      it relates to the Dwolla Platform?
+    </span>
+  }
+>
+  <ul>
+    <li>Send funds - No</li>
+    <li>
+      Receive funds - Yes - Note that funds will only process to their balance
+      and the transfer will stay <code>pending</code> until the Customer has
+      been verified.{" "}
+    </li>
+    <li>Add and verify a bank funding source - Yes</li>
+    <li>Add and verify a Beneficial Owner - Yes</li>
+  </ul>
+</Collapsible>
+
+<br />
+<br />
+
+<Collapsible
+  triggerText={
+    <span>
+      Q: My Customer has a <code>deactivated</code> or <code>suspended</code>{" "}
+      status. What activity would they able to engage in while being in{" "}
+      <code>deactivated</code> or <code>suspended</code> status, as it relates
+      to the Dwolla Platform?
+    </span>
+  }
+>
+  <ul>
+    <li>Send funds - No</li>
+    <li>Receive funds - No</li>
+    <li>Add and verify a bank funding source - No</li>
+    <li>Add and verify a Beneficial Owner - No</li>
+  </ul>
+</Collapsible>
+
+<br />
+<br />
+
+<Collapsible
+  triggerText={
+    <span>
+      Q: My Customer has a <code>verified</code> status, but is unable to send
+      funds. Why is this?
+    </span>
+  }
+>
+  <ul>
+    <li>
+      Your Customer has likely not completed the bank verification process. You
+      can check to see the status of the funding source{" "}
+      <a href="https://docs.dwolla.com/#list-funding-sources-for-a-customer">
+        via the API
+      </a>{" "}
+      or by going into the{" "}
+      <a href="https://www.dwolla.com/platform/dashboard">Dwolla dashboard</a>
+    </li>
+  </ul>
```

Comment:
For the answers that only have one bullet point, it looks like the designs have taken them out of list format: https://projects.invisionapp.com/d/main#/console/19172953/403040545/preview

Reply:
Ah yeah. I was going back and forth between both, since the markdowns had them as single list items. But I think `<p>`s indeed looks better here. This [commit](https://github.com/Dwolla/dev-portal/pull/82/commits/914fdc5b2c0ed82d3c421c285ea55d5f7595c9fe) should fix it!